18F/identity-idp
# LG-11600 confirm user has completed how to verify step

## 🎫 [Ticket](https://cm-jira.usa.gov/browse/LG-11600)

## 🛠 Summary of changes

Write a brief description of what you changed.

1. Added a new method to `idv_step_concern` to run as a before action on hybrid handoff and document capture pages.
2. This method will check that the `idv_session.skip_doc_auth` field is not nil
3. Specs added.
4. Small formatting change to key, wrapped in single quotes to align with styling in file.

## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

1. Ensure that both `in_person_proofing_enabled` and `in_person_proofing_opt_in_enabled` are set to true in your application.yml.default file
2. Navigate to /verify/how_to_verify
3. Do NOT select either of the radio buttons
4. Edit the url in the browser - navigate to /verify/hybrid_handoff
5. Confirm that you are bounced back to /verify/how_to_verify
6. Do NOT select either of the radio buttons 
7. Edit the url in the browser - navigate to /verify/document_capture
8. Confirm that you are bounced back to /verify/how_to_verify
9. Select one of the radio buttons and proceed.
